.github/workflows/test.yaml

```diff
@@ -27,10 +27,5 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-    # there is a problem in node 16 in CI
-    # which can't find eslint/eslint from dependencies
-    # so we install them globally
-    - run: npm install -g eslint
-    - run: npm install -g jest
```

Comment:
Removed because @panuhorsmalahti find the solution => regenerate pacakge-lock.json https://stackoverflow.com/a/63442044